### PR TITLE
Genomics workflows refinements

### PR DESCRIPTION
**Pull request by bedroesb**
- added tag and changed workflownames to match the others
- deleted double workflow: Genomics-4-parallel-download.g 
- updated all_covid_tools.yaml in genomics

### CODE REVIEW

File: genomics/deploy/workflows/Genomics-4-Variant_Calling_Lofreq.ga [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "a_galaxy_workflow": "true",
     "annotation": "",
     "format-version": "0.1",
-    "name": "COVID-19: Variant Calling Lofreq",
+    "name": "COVID-19 - Genomics [4] Variant Calling Lofreq",
```
Comment (mvdbeek) [inline genomics/deploy/workflows/Genomics-4-Variant_Calling_Lofreq.ga:5]: Please don't change the name. The ` - Genomics [4]` part doesn't make any sense outside of this repo.

**Review by mvdbeek — COMMENTED** [review]
Thanks. A heads up, these are going to migrate to https://github.com/iwc-workflows

File: genomics/deploy/workflows/Genomics-4-Variant_Calling_Lofreq.ga (hunk 1 above)
Comment (bedroesb) [reply genomics/deploy/workflows/Genomics-4-Variant_Calling_Lofreq.ga:5]: Maybe I change all the other workflow names too than ?

Comment (mvdbeek) [reply genomics/deploy/workflows/Genomics-4-Variant_Calling_Lofreq.ga:5]: 👍 I think that's a good idea

Comment (bedroesb) [reply genomics/deploy/workflows/Genomics-4-Variant_Calling_Lofreq.ga:5]: I will add genomics and cheminformatics maybe as tag, to have that as visual distinction if these workflows will get buried in a galaxy instance? Or are these short names of the workflows self explanatory enough?

Comment (mvdbeek) [reply genomics/deploy/workflows/Genomics-4-Variant_Calling_Lofreq.ga:5]: The tags sounds good!

Comment (bedroesb) [reply genomics/deploy/workflows/Genomics-4-Variant_Calling_Lofreq.ga]: @mvdbeek done!

**Review by bgruening — APPROVED** [review]

**Pull request merged**